include/fmt/format.h

```diff
@@ -3950,6 +3950,26 @@ class formatter<std::basic_string<Char, Traits, Allocator>, Char>
 template <typename Char, size_t N>
 struct formatter<Char[N], Char> : formatter<basic_string_view<Char>, Char> {};
 
+#if FMT_USE_BITINT
```

Comment:
Let's use SFINAE instead of conditional compilation.

Reply:
We probably could, but for example on gcc with `pedantic` it will error with:

```
include/fmt/format.h:3962:46: error: ISO C++ does not support ‘__int128’ for ‘type name’ [-Wpedantic]
 3962 |                     conditional_t<is_signed, __int128, unsigned __int128>,
```

I can disable the warning for that block etc. but not sure it's worthit, especially if msvc and other platforms may have other errors